nftserv/server.go

```diff
@@ -142,17 +150,22 @@ func (s *Server) handleNFTRequest(w http.ResponseWriter, r *http.Request, handle
 }
 
 func (s *Server) handlePUTnft(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Access-Control-Allow-Origin", "*")
-	if r.Method == http.MethodOptions {
-		return
-	}
-
 	var newtkn nft.NFT
 	if err := json.NewDecoder(r.Body).Decode(&newtkn); err != nil {
```

Comment:
Possible, with a less granular error message on the receiving side. We can simply limit the body to be of max size `maxTitleLength + maxDescriptionLength` (in bytes of course).
There are some nuances to this, but for the demo this would be enough and is stricter, than what is implemented currently. Could then also drop this message with a middleware :man_shrugging: if desired.

Reply:
Then I propose that we remove `max(Title|Description)Length` from the config and logic and change it to a simple `maxPayloadSize`. Then we can check they body's size before decoding it.